### DISCUSSION TITLE
How can you programmatically materialize an asset which is deployed to dagster cloud?

### DISCUSSION THREAD

**Discussion by dhirschfeld** [dagster-io/dagster#18877] [Q&A]
In dagster cloud I would just click on the materialize button in the asset view. Is there a way I can do that in code, via the GraphQL api (or otherwise)?

**Comment by dhirschfeld-ffma** [comment]
It looks like there's a feature request for this:
 - https://github.com/dagster-io/dagster-cloud/issues/13
 
 Does that mean it's not currently possible?
